### PR TITLE
User story1

### PR DESCRIPTION
**Pull request by deriancastro**
Here is my first user-story, thanks in advance for reviewing it. Note: in the Vercel preview, use the "Responsive Design Mode" in your Browser!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
+  const list = [
+    'Uchimata',
+    'Harai goshi',
+    'Seoi nage',
+    'Ko uchi gari',
+    'O soto gari',
+    'Kubi nage',
+    'Yoko tomoe',
+    'Kata guruma',
+  ]
```
Comment (anniviz) [inline src/App.js:14]: maybe you want to be more specific in naming the list, like "throwingsList" (if thats the right title 🤪)

File: src/components/Card.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+const Wrapper = styled.section`
+  display: grid;
+  align-content: center;
+  padding: 30px;
+  border-radius: 8px;
+  box-shadow: 0 4px 8px #0006;
+  background: white;
+`
+const Text = styled.p`
```
Comment (anniviz) [inline src/components/Card.js:24]: will this be a header of the card? Maybe you want it to be a `h3`or something similar

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <Title>{children}</Title>
+    </Wrapper>
+  )
+}
+
+const Title = styled.h2`
+  text-align: center;
+  margin: 0;
+`
+const Wrapper = styled.section`
```
Comment (anniviz) [inline src/components/Header.js:20]: do you need the Wrapper element or can you add the styling to the Title element?

**Review by anniviz — COMMENTED** [review]
nice work!
just a few tiny things you might want to consider.

File: src/App.js (hunk 1 above)
Comment (deriancastro) [reply src/App.js:14]: It's a very good idea, thank you!

File: src/components/Card.js (hunk 2 above)
Comment (deriancastro) [reply src/components/Card.js:24]: I want in my letter only the name of the technique and when I click on it, it opens another page with a picture of the technique and the header of the page has the name of the technique.
but I think you are right :)

File: src/components/Header.js (hunk 3 above)
Comment (deriancastro) [reply src/components/Header.js:20]: Ok, I was thinking about the next user-stories... then I will have a log out button in the Header, but for now I don't need a Wrapper. Thanks for your review!

**Pull request merged**